src/openms/source/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.cpp

```diff
@@ -343,7 +343,7 @@ namespace OpenMS
 
         if ((String)feature_it.getMetaValue("id_target_num_transitions") != "")
         {
-          int id_target_num_transitions = feature_it.getMetaValue("id_target_num_transitions").toString().toInt();
+          int id_target_num_transitions = std::stoi(feature_it.getMetaValue("id_target_num_transitions").toString(),nullptr);
```

Comment:
@cbielow @timosachsenberg can you maybe comment on this, this looks really ugly and I was under the impression / hope that `toInt()` would "just work"

Reply:
I think it did before, but this seems to be broken now. Were there any changes recently?

Reply:
not in `toInt()` but in how floats and integers are streamed to strings I think

Reply:
seems relatively well tested: https://github.com/OpenMS/OpenMS/blob/develop/src/tests/class_tests/openms/source/String_test.cpp#L541

Reply:
Ok, I found the issue. ``num_transitions`` is added as ``Score`` (double) and not as ``MetaValue`` and thus returned ``double``.